### PR TITLE
Add delete views instructions

### PR DESCRIPTION
**Pull request by mikaelarguedas**
This alse fixes the style so that the new section can be displayed correctly

### CODE REVIEW

File: doc/ongoing_operations.rst [hunk 1]
```diff
@@ -134,9 +134,32 @@ Disable all jobs related to a ROS distribution
...
- * ``Lrel_`` which matches the Lunar release related management jobs.
- * ``Ldev_`` which matches the Lunar devel jobs as well as the management related jobs.
- * ``Ldoc_`` which matches the Lunar doc jobs as well as the management related jobs.
- * ``Lpr_`` which matches the Lunar PR jobs as well as the management related jobs.
+* ``Lsrc_`` which matches all Lunar source jobs.
+* ``Lbin_`` which matches all Lunar binary jobs.
+* ``Lrel_`` which matches the Lunar release related management jobs.
+* ``Ldev_`` which matches the Lunar devel jobs as well as the management related jobs.
+* ``Ldoc_`` which matches the Lunar doc jobs as well as the management related jobs.
+* ``Lpr_`` which matches the Lunar PR jobs as well as the management related jobs.
```
Comment (dirk-thomas) [inline doc/ongoing_operations.rst]: Please avoid unrelated whitespace changes like this. All other files use a space to indent the bullets.

File: doc/ongoing_operations.rst [hunk 2]
```diff
@@ -134,9 +134,32 @@ Disable all jobs related to a ROS distribution
...
+
+Deleting all views related to a ROS distribution
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you deleted all the jobs of a given ROS distribution, noe all the views associated with them are empty.
```
Comment (dirk-thomas) [inline doc/ongoing_operations.rst]: `noe`?

File: doc/ongoing_operations.rst [hunk 3]
```diff
@@ -134,9 +134,32 @@ Disable all jobs related to a ROS distribution
...
+You can delete them manually by going to specific view and click the "Delete View" button on the left sidebar.
```
Comment (dirk-thomas) [inline doc/ongoing_operations.rst]: > going to specific view

Missing "the"?

File: doc/ongoing_operations.rst [hunk 4]
```diff
@@ -134,9 +134,32 @@ Disable all jobs related to a ROS distribution
...
+Or programmatically, using the same prefixes as the ones use to delete the jobs:
```
Comment (dirk-thomas) [inline doc/ongoing_operations.rst]: > as the ones use to delete

"used"?

File: doc/ongoing_operations.rst (hunk 1 above)
Comment (mikaelarguedas) [reply doc/ongoing_operations.rst]: Without this change the bullet list is displayed as commented out. That doesn't seem to be what we want here. I can submit it as a separate PR if you prefer

**Comment by mikaelarguedas** [comment]
thanks for homogenizing the bullet lists
